reference/checkouts.v3.yml

```diff
@@ -88,6 +88,49 @@ paths:
         AllowDynamicQueryParameters: false
         AllowDynamicFormParameters: false
         IsMultiContentStreaming: false
+    put:
+      description: |-
+        Change customer message to an existing *Checkout*.
+      tags:
+        - Checkout
+      operationId: CheckoutsByCheckoutIdPut
+      produces:
+        - application/json
+      parameters:
+        - name: checkoutId
+          in: path
+          type: string
+          required: true
+          description: Id of the Checkout
+        - name: Accept
+          in: header
+          type: string
+          default: application/json
+          required: true
+        - name: Content-Type
+          in: header
+          type: string
+          default: application/json
+          required: true
+        - name: body
+          in: body
+          required: true
+          description: customer_message is required.
```

Comment:
```suggestion
          description: `customer_message` is required.
```

Reply:
changed. but a little different, use: the `customer_message` is required. because the first character of value cannot be "`".

<img width="618" alt="Screen Shot 2021-12-29 at 9 33 59 am" src="https://user-images.githubusercontent.com/94653751/147612746-2107e0ba-a765-4d55-a1ae-fc1f70eb1181.png">